cachi2/core/package_managers/bundler/main.py

```diff
@@ -13,6 +13,7 @@
 from cachi2.core.models.sbom import Component
 from cachi2.core.package_managers.bundler.parser import (
     GemPlatformSpecificDependency,
+    GitDependency,
```

Comment:
nitpick: Do we really want to start the commit subject with "bug" :) ? It looks like you're introducing one, not fixing it.

Reply:
I might be. Any piece of code could contain a bug or enable a previously hidden bug. Starting every commit message with `bug:` would make us explicitly acknowledge this.

Reply:
> I might be. Any piece of code could contain a bug or enable a previously hidden bug. Starting every commit message with `bug:` would make us explicitly acknowledge this.

Amen! :D